app/backend/src/couchers/templates/v2.py

```diff
@@ -96,6 +96,17 @@ def v2markdown(value: str) -> str:
     return md.render(value)  # type: ignore[no-any-return]
 
 
+def replace_tag(match: re.Match[str]) -> str:
```

Comment:
Does mypy have issues with local functions?

Reply:
But to answer your question: mypy is fine with local functions, but performance-wise, it's better to avoid those. Usually it doesn't matter, but if the templating function is called many times during a single template rendering, it can lead to a couple of ms lost. It's tiny, but why lose performance if you can avoid it without sacrificing anything